.github/ISSUE_TEMPLATE/add_content.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Add content request
+about: Suggestion of topic to be added to an existing page of the Infectious Diseases Toolkit
```

Comment:
This does not sound like a correct English sentence or is it just me? :) Should it be Topic Suggestion for an existing page or topic to be added to an existing page or ?